config/src/main/java/org/hyperledger/besu/config/StubGenesisConfigOptions.java

```diff
@@ -266,9 +282,8 @@ public OptionalLong getEcip1017EraRounds() {
     getIstanbulBlockNumber().ifPresent(l -> builder.put("istanbulBlock", l));
     getMuirGlacierBlockNumber().ifPresent(l -> builder.put("muirGlacierBlock", l));
     getBerlinBlockNumber().ifPresent(l -> builder.put("berlinBlock", l));
-    // TODO EIP-1559 change for the actual fork name when known
-    getEIP1559BlockNumber().ifPresent(l -> builder.put("aleutblock", l));
-
+    getLondonBlockNumber().ifPresent(l -> builder.put("londonBlock", l));
```

Comment:
do we not also need baikalBlockNumber?  or are you thinking we will just going to use londonBlock only and keep aleut for backward compatibility?

Reply:
No we did this for the Berlin testnets (yoloBlock) . In the genesis we will have baikalBlock and internally we consider it as a londonBlocK. We will remove this mechanism after testing.

For aleut we need it because, for now, is different from London. Aleut is a testnet with just EIP1559 and BASEFEE while baikal is a testnet that wants to be like london (with all EIPS).

For now for retro compatibility I prefer to keep it like that. To avoid consensus issue in this testnet